lib/plug/static.ex

```diff
@@ -112,43 +112,43 @@ defmodule Plug.Static do
   end
 
   def init(opts) do
-    at     = Keyword.fetch!(opts, :at)
-    from   = Keyword.fetch!(opts, :from)
-    gzip   = Keyword.get(opts, :gzip, false)
-    brotli = Keyword.get(opts, :brotli, false)
-    only   = Keyword.get(opts, :only, [])
+    at = Keyword.fetch!(opts, :at)
+    from = Keyword.fetch!(opts, :from)
+    gzip? = Keyword.get(opts, :gzip, false)
+    brotli? = Keyword.get(opts, :brotli, false)
+    only = Keyword.get(opts, :only, [])
     prefix = Keyword.get(opts, :only_matching, [])
```

Comment:
I've removed whitespace for the very practical reason: unfortunately, it always makes code to rot – not once we've end up with broken indentation, also it brings needless noise in git diff (blame too) which makes harder to follow pure changes on the lines.

In this situation, I have added `?` to `brotli` what brings, in turn, the need to add extra space to 5 more lines around to keep whitespace correct, thus git diff noise and broken blame is here; I decided to remove whitespace instead.

Of course, if everyone else likes idea to keep maintaining whitespace I will alter this the other way around.